core/src/main/java/org/apache/accumulo/core/clientImpl/RootClientTabletCache.java

```diff
@@ -87,20 +99,24 @@ public List<Range> findTablets(ClientContext context, List<Range> ranges,
   }
 
   @Override
-  public void invalidateCache(KeyExtent failedExtent) {}
+  public void invalidateCache(KeyExtent failedExtent) {
+    // no-op see class level javadoc
+  }
 
   @Override
-  public void invalidateCache(Collection<KeyExtent> keySet) {}
+  public void invalidateCache(Collection<KeyExtent> keySet) {
+    // no-op see class level javadoc
+  }
 
   @Override
   public void invalidateCache(ClientContext context, String server) {
-    ZooCache zooCache = context.getZooCache();
-    String root = context.getZooKeeperRoot() + Constants.ZTSERVERS;
```

Comment:
This seems like a bug.  Its clearing something that has nothing to do w/ the root tablet metadata stored in zookeeper.  So removed it.  It probably not really hurting anything.  Thought about back porting a change to remove these lines in earlier versions.